plugins/home/src/homePageComponents/WelcomeTitle/timeUtil.ts

```diff
@@ -22,7 +22,10 @@ import goodEvening from './locales/goodEvening.locales.json';
 // every time the user navigates.
 const greetingRandomSeed = Math.floor(Math.random() * 1000000);
 
-export function getTimeBasedGreeting(): { language: string; greeting: string } {
+export function getTimeBasedGreeting(language?: string): {
```

Comment:
There are tests for this component so we should be adding new ones to cover this new case. Simply add them to the `WelcomeTitle.text.tsx` file, there are lots of examples in the code base to get you going

Reply:
added test case for multi language